geometry_msgs/Vector3 position  # meters from origin
geometry_msgs/Vector3 velocity  # meters per second
